Week2/homework-midy/collectiveAge.js

```diff
@@ -0,0 +1,19 @@
+const hackYourFutureMembers = [
+  { name: 'Wouter', age: 33 },
+  { name: 'Federico', age: 32 },
+  { name: 'Noer', age: 27 },
+  { name: 'Tjebbe', age: 22 },
+  { name: 'Bjorne', age: 24 },
+];
+
+function combineAges(people, combiner) {
+  const ages = people.map(obj => obj.age);
+  return `The collective age of the HYF team is:\n${combiner(ages)}`;
+}
+
+function add(ages) {
+  const addedAges = ages.reduce((a, b) => a + b);
+  return addedAges;
+}
```

Comment:
Very good! One side note
the `addedAges` can be omitted here. You can do 
```js
function add(ages) {
  return ages.reduce((a, b) => a + b);;
}
or 
const add = (ages) => ages.reduce((a, b) => a + b);

Reply:
Is using one-liners better than using a `function`?

Reply:
In case of one liners, yes it is better to use arrow function. I use functions in order to describe the functionality with a name. eg `function sortByName () {` now I know what this function does without reading the code inside it